ModAssistant/Localisation/de.xaml

```diff
@@ -19,6 +19,7 @@
     <sys:String x:Key="MainWindow:ModsButton">Mods</sys:String>
     <sys:String x:Key="MainWindow:AboutButton">Über</sys:String>
     <sys:String x:Key="MainWindow:OptionsButton">Optionen</sys:String>
+    <sys:String x:Key="MainWindow:PlayButton">Starten!</sys:String>
```

Comment:
You should change this to `Spielen!` to be consistent with `NL` or `EN`.

| EN   | DE  |
|---|---|
| Play  | Spielen  |
| Start  | Starten  |

```suggestion
    <sys:String x:Key="MainWindow:PlayButton">Spielen!</sys:String>
```